blosc2-sys/build.rs

```diff
@@ -1,25 +1,37 @@
 use std::path::PathBuf;
 
 fn main() {
-    let lib = cmake::Config::new("c-blosc2")
-        .define("CMAKE_POSITION_INDEPENDENT_CODE", "ON")
-        .define("CMAKE_C_FLAGS", "-fPIE")
-        .define("BLOSC_INSTALL", "ON")
-        .define("BUILD_TESTS", "OFF")
-        .define("BUILD_EXAMPLES", "OFF")
-        .define("BUILD_BENCHMARKS", "OFF")
-        .define("BUILD_FUZZERS", "OFF")
-        .define("STATIC_LIB", "ON")
-        .define("SHARED_LIB", "ON")
-        .build();
+    // let lib = cmake::Config::new("c-blosc2")
+    //     .define("CMAKE_C_FLAGS", "-O3 -std=c99")
+    //     .define("STATIC_LIB", "ON")
+    //     .define("SHARED_LIB", "ON")
+    //     .define("BUILD_TESTS", "OFF")
+    //     .define("BUILD_EXAMPLES", "OFF")
+    //     .define("BUILD_BENCHMARKS", "OFF")
+    //     .define("BUILD_FUZZERS", "OFF")
+    //     .define("CMAKE_POSITION_INDEPENDENT_CODE", "ON")
+    //     .define("BLOSC_INSTALL", "ON")
+    //     .always_configure(true)
+    //     .build();
 
-    println!("cargo:rustc-link-search={}/lib64", lib.display());
-    println!("cargo:rustc-link-search={}/lib", lib.display());
+    // println!("cargo:rustc-link-search=/home/milesg/mambaforge/envs/cramjam/lib64");
+    println!(
+        "cargo:rustc-link-search={}/c-blosc2/install/lib64",
+        env!("CARGO_MANIFEST_DIR")
+    );
```

Comment:
Basically need to just change the comments between these two lines to run benchmarks again to compare. `blosc2` just happens to be installed in my mambaforge env, so would need to adjust that and the locally built lib in the submodule in `install`